ui/src/tool-panel/km-post/km-post-infobox.tsx

```diff
@@ -166,16 +166,19 @@ const KmPostInfobox: React.FC<KmPostInfoboxProps> = ({
                     />
                 </InfoboxContent>
             </Infobox>
-            {kmPost.draftType !== 'NEW_DRAFT' && (
-                <AssetValidationInfoboxContainer
-                    contentVisible={visibilities.validation}
-                    onContentVisibilityChange={() => visibilityChange('validation')}
-                    id={kmPost.id}
-                    type={'KM_POST'}
-                    publishType={publishType}
-                    changeTime={kmPostChangeTime}
-                />
-            )}
+            {
+                // TODO: GVT-2522
```

Comment:
En vielä poistanut näitä validointicheckejä koska haluan tuon tiketin fixit omalle pullarille/commitille. Teen sen seuraavaksi. Nykyversion pitäisi toimia niinkuin edeltäjänsä